src/ucp/rndv/rndv.c

```diff
@@ -159,7 +159,7 @@ size_t ucp_rndv_rts_pack(ucp_request_t *sreq, ucp_rndv_rts_hdr_t *rndv_rts_hdr,
         packed_rkey_size      = ucp_rkey_pack_uct(
                 worker->context, sreq->send.state.dt.dt.contig.md_map,
                 sreq->send.state.dt.dt.contig.memh, &mem_info,
-                ucp_ep_config(sreq->send.ep)->uct_rkey_pack_flags, 0, NULL,
```

Comment:
bugfix found by the check?
do we need to port it to v1.13.x?

Reply:
> bugfix found by the check?

yes
> do we need to port it to v1.13.x?

yes